### PR TITLE
Depreciated unused variable from Integration tests

### LINKED ISSUE #2110: Remove unused variable from integration tests
Simple task, just need to remove this unused variable from the integration tests:

https://github.com/ethereum-optimism/optimism/blob/137f7765b87c862256f0b847d40e18f597895593/integration-tests/test/shared/utils.ts#L148-L149

This variable is used within the `native-eth.spec.ts` test file to get a `sequencerBalance`, but the `sequencerBalance` variable is never actually used after that:

https://github.com/ethereum-optimism/optimism/blob/137f7765b87c862256f0b847d40e18f597895593/integration-tests/test/native-eth.spec.ts#L34-L36

So we can just remove `PROXY_SEQUENCER_ENTRYPOINT_ADDRESS` and `sequencerBalance`. 

---

Please remember to read the [Contributing Guidelines](https://github.com/ethereum-optimism/optimism/blob/develop/CONTRIBUTING.md) if this is your first time contributing to Optimism! It contains useful information that will make the contributing process as easy as possible.

### PR DESCRIPTION
**Pull request by RishiKumarRay**
**Description**
 remove the unused variable from the integration tests

**Metadata**
- Fixes #2110

### CODE REVIEW

File: integration-tests/test/native-eth.spec.ts [hunk 1]
```diff
@@ -31,9 +31,6 @@ describe('Native ETH Integration Tests', async () => {
     const l1BobBalance = await l1Bob.getBalance()
     const l2BobBalance = await l2Bob.getBalance()
 
-    const sequencerBalance = await _env.ovmEth.balanceOf(
```
[the remark this answers is not in the document]
Comment (RishiKumarRay) [reply integration-tests/test/native-eth.spec.ts:34]: @smartcontracts i have removed it, please review and thanks for letting me know this

**Comment by RishiKumarRay** [comment]
@smartcontracts i saw a proxy sequencer reference that was not removed and it was failing linting workflow but now i have removed that, can you trigger please again

**Comment by RishiKumarRay** [comment]
@smartcontracts looks like everything has passed ? Is there anything else ?

**Comment by smartcontracts** [comment]
@RishiKumarRay would you mind [rebasing](https://github.com/ethereum-optimism/optimism/blob/develop/CONTRIBUTING.md#rebasing) this PR into a single commit using the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format? Once that's done, I'll go ahead and approve (and then merge).

**Comment by smartcontracts** [comment]
@RishiKumarRay let me know if you're having any trouble rebasing, I can try to help

**Comment by RishiKumarRay** [comment]
@smartcontracts i have messed up squashing the commit , can you guide me please?

**Comment by RishiKumarRay** [comment]
@smartcontracts please check the rebase , i think i have successfully done it

**Comment by RishiKumarRay** [comment]
@smartcontracts Since i can see now only one commit is present so it is done right?

**Comment by smartcontracts** [comment]
> @smartcontracts Since i can see now only one commit is present so it is done right?

Yep! Perfect. I will run the integration tests one more time...

**Comment by RishiKumarRay** [comment]
@smartcontracts Thanks , the tests has passed too

**Review by smartcontracts — APPROVED** [review]

**Comment by smartcontracts** [comment]
Great work, thank you @RishiKumarRay! Feel free to email me at [kelvin AT optimism DOT io] for a hand-drawn contributor NFT :-)

**Comment by RishiKumarRay** [comment]
@smartcontracts thanks , this is cool , i really want something like that , also i have sended you mail from my personal id - rishirai 760  AT gmail dot com, you might have received it. thanks

**Comment by RishiKumarRay** [comment]
@smartcontracts did you received my email??

**Pull request merged**